tests/Doctrine/Tests/ORM/Tools/Export/AbstractClassMetadataExporterTest.php

```diff
@@ -371,6 +373,26 @@ public function testCascadeAllCollapsed()
         }
     }
 
+    /**
+     * @depends testExportedMetadataCanBeReadBackIn
```

Comment:
That's how all other export tests are structured, so I just adhered to the structure within the test case.